src/stylesheets/common/_lists.scss

```diff
@@ -0,0 +1,39 @@
+article {
```

Comment:
Gotcha.  I think that was the part of the conversation that Meghan and I missed.  Darn audio.